modules/database/outputs.tf

```
@@ -1,15 +1,23 @@
 output "db_endpoint" {
   value = aws_db_instance.postgresql.endpoint
+
+  description = "The connection endpoint of the PostgreSQL RDS instance, in address:port format."
```

Minor change, but I don't think the comma is needed. 

```suggestion
  description = "The connection endpoint of the PostgreSQL RDS instance in address:port format."
```